### ISSUE TITLE
Migrate domains blocked by __cfduid cookies.

### ISSUE THREAD

**Issue by cowlicks** [EFForg/privacybadger#1538]
User's might still have many sites blocked by the cloudflare `__cfduid` cookie. We should unblock these.

**Comment by ghostwords** [comment]
Following up on https://github.com/EFForg/privacybadger/issues/1533#issuecomment-318782365.

We should first try to figure out if this is a thing and how big of a thing. One approach idea: Of all the domains in error reports/GitHub issues submitted using version `2017.5.9` (when we started ignoring `__cfduid`) or later, how many domains do nothing except set/get `__cfduid` cookies?

**Comment by cowlicks** [comment]
Instead of doing that, we could look at blocked domains, then check if these have *only* a `__cfduid` cookie, if that is the case the domain should be unblocked. Regardless of what error reports we have. If there is another reason it was blocked,  Privacy Badger will eventually block it again.

**Comment by ghostwords** [comment]
This sounds reasonable, as an implementation approach. We should get a number of specific cases to verify the migration though. Can that cookie db site (`https://cookiepedia.co.uk/`?) help?

**Comment by cowlicks** [comment]
From [here](https://cookiepedia.co.uk/cookies/__cfduid).

```

About this cookie:
Cookie assoiated with sites using CloudFlare, used to speed up page load times. According to CloudFlare it is used to override any security restrictions based on the IP address the visitor is coming from. It does not contain any user identification information.

The main purpose of this cookie is: Strictly Necessary

Key numbers for __cfduid:
Cookies with this name have been found on 14,461 websites, set by 9,650 host domains.

It has been found as a First Party cookie on 5,466 websites and a Third Party cookie on 18,290 websites.

It has been found as a Persistent cookie on 23,740 websites, with an average life span of 2,262 days.

It has been found as a Session cookie on 16 websites.
```

**Comment by ghostwords** [comment]
Get the full list of blocked domains from Privacy Badger versions after the Cloudflare workaround went out:
```sql
DROP TABLE IF EXISTS numbers; CREATE TEMPORARY TABLE numbers AS (SELECT id FROM reports WHERE id <= (SELECT MAX(ROUND((LENGTH(block) - LENGTH(REPLACE(block, ",", ""))) / LENGTH(",")) + 1) AS max_split_length FROM reports WHERE (version = "2017.5.9" OR version = "2017.6.13" OR version = "2017.6.13.1" OR version = "2017.7.24" OR version = "2017.9.12" OR VERSION = "2017.9.12.1"))); SELECT blocked_fqdn FROM (SELECT SUBSTRING_INDEX(SUBSTRING_INDEX(reports.block, ",", numbers.id), ",", -1) AS blocked_fqdn FROM numbers INNER JOIN reports ON CHAR_LENGTH(reports.block) - CHAR_LENGTH(REPLACE(reports.block, ",", "")) >= numbers.id-1 WHERE (version = "2017.5.9" OR version = "2017.6.13" OR version = "2017.6.13.1" OR version = "2017.7.24" OR version = "2017.9.12" OR VERSION = "2017.9.12.1")) AS tmp GROUP BY blocked_fqdn;
```
Make a GET to "/" of each domain and see if `__cfduid` is in the response headers:
```bash
for domain in $(cat domains.txt); do curl -Iv -m10 "$domain" 2>&1 | grep -q __cfduid && echo "$domain"; done | sort
```

**Comment by ghostwords** [comment]
Here is the list of Cloudflare-using domains reported as blocked in errors reports from Badgers version `2017.5.9` and above, ordered it by prevalence:
```
+-------+-----------------------------------------+
| count | blocked_fqdn                            |
+-------+-----------------------------------------+
|   326 | cdn-images-1.medium.com                 |
|   286 | cdn-static-1.medium.com                 |
|    98 | tru.am                                  |
|    96 | cdn.onesignal.com                       |
|    76 | edge.alluremedia.com.au                 |
|    72 | cdn.viglink.com                         |
|    63 | rum-static.pingdom.net                  |
|    51 | www.lightboxcdn.com                     |
|    49 | www.dianomi.com                         |
|    37 | cdn.inspectlet.com                      |
|    33 | browser-update.org                      |
|    31 | www.npttech.com                         |
|    30 | static.adzerk.net                       |
|    29 | try.abtasty.com                         |
|    28 | a.fsdn.com                              |
|    28 | static.addtoany.com                     |
|    27 | experience.tinypass.com                 |
|    26 | prebid.districtm.ca                     |
|    24 | p.bm23.com                              |
|    23 | cdn.tinypass.com                        |
|    22 | js.bronto.com                           |
|    18 | widget.uservoice.com                    |
|    18 | cdn.pbbl.co                             |
|    17 | stats.lotlinx.com                       |
|    14 | geoservice.curse.com                    |
|    14 | freegeoip.net                           |
|    12 | cdn.pubexchange.com                     |
|    11 | tag.navdmp.com                          |
|    11 | secure.statcounter.com                  |
|    11 | domain157.club                          |
|    11 | cdn.tynt.com                            |
|    10 | static.mijnwebwinkel.nl                 |
|    10 | asset.mijnwebwinkel.nl                  |
|    10 | analytics.codigo.se                     |
|    10 | files.imbox.io                          |
|     9 | cdn.districtm.ca                        |
|     9 | c.statcounter.com                       |
|     9 | rangeblessedness.men                    |
|     9 | www.marinetraffic.com                   |
|     8 | static.impresa.pt                       |
|     8 | dashboard.tinypass.com                  |
|     8 | cdn.datatables.net                      |
|     8 | onesignal.com                           |
|     8 | westen-r.life                           |
|     7 | imageproxy.ivaws.com                    |
|     7 | thumb.oloadcdn.net                      |
|     7 | c.lytics.io                             |
|     7 | thumbnailer.mixcloud.com                |
|     7 | ghbtns.com                              |
|     7 | images-cdn.impresa.pt                   |
|     7 | secure.apps.shappify.com                |
|     6 | c.shpg.org                              |
|     6 | st.prntscr.com                          |
|     6 | prntscr.com                             |
|     6 | lucyassets.abcdn.nl                     |
|     6 | snippet.minute.ly                       |
|     6 | api.tinypass.com                        |
|     6 | www.justuno.com                         |
|     6 | api.trackuity.com                       |
|     6 | tcr.tynt.com                            |
|     6 | codepen.io                              |
|     6 | image.tmdb.org                          |
|     6 | main.pubexchange.com                    |
|     6 | i.stack.imgur.com                       |
|     6 | streamango.com                          |
|     5 | app.clickfunnels.com                    |
|     5 | am.medialaben.no                        |
|     5 | cdn.nextopia.net                        |
|     5 | js.appboycdn.com                        |
|     5 | hsto.org                                |
|     5 | www.thevideo.me                         |
|     5 | embed.tawk.to                           |
|     5 | www.insticator.com                      |
|     5 | api-cdn.embed.ly                        |
|     5 | fea.assettype.com                       |
|     5 | www.ratecity.com.au                     |
|     5 | settings.luckyorange.net                |
|     5 | app.backinstock.org                     |
|     5 | top-drawer-c.life                       |
|     4 | images.clickfunnels.com                 |
|     4 | dsg.btttag.com                          |
|     4 | static.hgmsites.net                     |
|     4 | images.hgmsites.net                     |
|     4 | csp.medium.com                          |
|     4 | app1assets.clickfunnels.com             |
|     4 | images.abcdn.nl                         |
|     4 | istatic.davebestdeals.com               |
|     4 | img.shields.io                          |
|     4 | nir.regmedia.co.uk                      |
|     4 | regmedia.co.uk                          |
|     4 | cdn.okccdn.com                          |
|     4 | cdn.engine.4dsply.com                   |
|     4 | assets-9gag-fun.9cache.com              |
|     4 | miscmedia-9gag-fun.9cache.com           |
|     4 | js-webt-lol.9cache.com                  |
|     4 | medium.com                              |
|     4 | www.shopperapproved.com                 |
|     4 | storage.proboards.com                   |
|     4 | tortoise.proboards.com                  |
|     4 | ads.2mdnsys.com                         |
|     3 | assets.clickfunnels.com                 |
|     3 | cdn.engine.phn.doublepimp.com           |
|     3 | shopify.privy.com                       |
|     3 | assets.hubstatic.com                    |
|     3 | appassets.clickfunnels.com              |
|     3 | www.affiliatly.com                      |
|     3 | rawgit.com                              |
|     3 | api-img.billiger.de                     |
|     3 | d.btttag.com                            |
|     3 | v2assets.zopim.io                       |
|     3 | habracdn.net                            |
|     3 | includes.okccdn.com                     |
|     3 | adyoulike.omnitagjs.com                 |
|     3 | s.shpg.org                              |
|     3 | shpg.org                                |
|     3 | cdn.impresa.pt                          |
|     3 | widget.privy.com                        |
|     3 | static.lpcdn.ca                         |
|     3 | fasttechcdn.com                         |
|     3 | img.fasttechcdn.com                     |
|     3 | img-9gag-fun.9cache.com                 |
|     3 | sw.graphiq.com                          |
|     3 | s.sloyalty.com                          |
|     3 | atm.impresa.pt                          |
|     3 | id.impresa.pt                           |
|     3 | buy.tinypass.com                        |
|     3 | st.bebi.com                             |
|     3 | cdm.cursecdn.com                        |
|     3 | b0.thejournal.ie                        |
|     3 | c3.thejournal.ie                        |
|     3 | c1.thejournal.ie                        |
|     3 | c2.thejournal.ie                        |
|     3 | bench-live.men                          |
|     2 | fogl1onf.com                            |
|     2 | o.mlv-cdn.com                           |
|     2 | s.mlv-cdn.com                           |
|     2 | j.wfcdn.de                              |
|     2 | scr.wfcdn.de                            |
|     2 | i.wfcdn.de                              |
|     2 | stmedia.stimg.co                        |
|     2 | cdn.augur.io                            |
|     2 | cdn01.androidauthority.net              |
|     2 | cdn03.androidauthority.net              |
|     2 | assets0.uvcdn.com                       |
|     2 | a.smedata.sk                            |
|     2 | code.ionicframework.com                 |
|     2 | cdn.trafficstars.com                    |
|     2 | widget.mixcloud.com                     |
|     2 | k3.okccdn.com                           |
|     2 | cdn.knightlab.com                       |
|     2 | beacon.errorception.com                 |
|     2 | assets-comment-fun.9cache.com           |
|     2 | pb.s3wfg.com                            |
|     2 | rsrc.s3wfg.com                          |
|     2 | derpicdn.net                            |
|     2 | images.mmcloud.se                       |
|     2 | st3.igstatic.com                        |
|     2 | st1.igstatic.com                        |
|     2 | st2.igstatic.com                        |
|     2 | cdn1.graphiq.com                        |
|     2 | static.zotabox.com                      |
|     2 | projects.elitechnology.com              |
|     2 | api.frankerfacez.com                    |
|     2 | rum-collector.pingdom.net               |
|     2 | easylist.club                           |
|     2 | imgw.rgcdn.nl                           |
|     2 | puu.sh                                  |
|     2 | images.lpcdn.ca                         |
|     2 | istatic.eshopcomp.com                   |
|     2 | realocdn.com                            |
|     2 | www.bungie.net                          |
|     2 | vidstreaming.io                         |
|     2 | tms.truoptik.com                        |
|     2 | img.dxcdn.com                           |
|     2 | seal.globalsign.com                     |
|     2 | d3.btttag.com                           |
|     2 | by2.uservoice.com                       |
|     2 | static.mailerlite.com                   |
|     2 | pstatic.davebestdeals.com               |
|     2 | api.impresa.pt                          |
|     2 | jornaldenoticias.os.tc                  |
|     2 | srv.bebi.com                            |
|     2 | image.prntscr.com                       |
|     2 | production-blaze-assets.ratecity.com.au |
|     2 | homedepot.btttag.com                    |
|     2 | dxj1e0bbbefdtsyig.woldrssl.net          |
|     2 | gscdn.nl                                |
|     2 | v2.zopim.com                            |
|     2 | i.embed.ly                              |
|     2 | c0.thejournal.ie                        |
|     2 | cdn.visadd.com                          |
|     2 | vuukle.com                              |
|     2 | networkcheck.xyz                        |
|     2 | media.saznajnovo.com                    |
|     2 | assets.adsttc.com                       |
|     2 | images.adsttc.com                       |
|     2 | afd.adsttc.com                          |
|     2 | www.projectwonderful.com                |
|     2 | tt1.openload.co                         |
|     2 | mellowads.com                           |
|     2 | media.sandhills.com                     |
|     2 | ihg.btttag.com                          |
|     2 | butstrap.space                          |
|     2 | cdn.optitc.com                          |
|     2 | hi.districtm.ca                         |
|     2 | spidtest.space                          |
|     2 | top-drawer-b.life                       |
|     2 | glyph.medium.com                        |
|     2 | netanalyzer.space                       |
|     2 | getstencil.com                          |
|     1 | pol.is                                  |
|     1 | mediaserver.emol.cl                     |
|     1 | gcdn.emol.cl                            |
|     1 | 20.theladbiblegroup.com                 |
|     1 | staging.20.65twenty.com                 |
|     1 | shared.65twenty.com                     |
|     1 | newschoolers.grindnetworks.com          |
|     1 | pamplinmedia.com                        |
|     1 | cdn.prizma.tv                           |
|     1 | cssjs.chesscomfiles.com                 |
|     1 | images.chesscomfiles.com                |
|     1 | www.janjuaplayer.com                    |
|     1 | worldnaturenet.xyz                      |
|     1 | urlvalidation.com                       |
|     1 | cdn.staticneo.com                       |
|     1 | iculture.textopus.nl                    |
|     1 | sanet.pics                              |
|     1 | www.mijnwebwinkel.nl                    |
|     1 | www.speedcheckercdn.com                 |
|     1 | api.peer5.com                           |
|     1 | api.w00f.net                            |
|     1 | k2.okccdn.com                           |
|     1 | k1.okccdn.com                           |
|     1 | k0.okccdn.com                           |
|     1 | twt-assets.washtimes.com                |
|     1 | twt-thumbs.washtimes.com                |
|     1 | kjarninn.overcastcdn.com                |
|     1 | totalstatic.com                         |
|     1 | player.gfrvideo.com                     |
|     1 | img1.s3wfg.com                          |
|     1 | img.s3wfg.com                           |
|     1 | img6.s3wfg.com                          |
|     1 | img4.s3wfg.com                          |
|     1 | img4.demotywatoryfb.pl                  |
|     1 | siteimproveanalytics.com                |
|     1 | purplestats.com                         |
|     1 | assets1.uvcdn.com                       |
|     1 | secure.surfbuyermac.com                 |
|     1 | opensharecount.com                      |
|     1 | fftf.io                                 |
|     1 | queue.fightforthefuture.org             |
|     1 | stfront1.igstatic.com                   |
|     1 | stfront2.igstatic.com                   |
|     1 | stfront3.igstatic.com                   |
|     1 | space.onesignal.com                     |
|     1 | app.newsharecounts.com                  |
|     1 | bcm.interactives.dk                     |
|     1 | images.interactives.dk                  |
|     1 | cdn02.androidauthority.net              |
|     1 | cdn-images-2.medium.com                 |
|     1 | twitchemotes.com                        |
|     1 | s.reembed.com                           |
|     1 | www.smartlinks.dianomi.com              |
|     1 | www.emg-network.com                     |
|     1 | ads.emg-network.com                     |
|     1 | ssl.widgets.webengage.com               |
|     1 | s0.cdn3x.com                            |
|     1 | t0.cdn3x.com                            |
|     1 | new4.fjcdn.com                          |
|     1 | new.fjcdn.com                           |
|     1 | cimg2.ibsrv.net                         |
|     1 | www.bitdefender.com                     |
|     1 | triggers.imbox.io                       |
|     1 | js.knowlead.io                          |
|     1 | w3p.checktrax.com                       |
|     1 | bitsheakers.utop.it                     |
|     1 | pixelpurehat.omnitagjs.com              |
|     1 | api.openload.co                         |
|     1 | bryantrv.com                            |
|     1 | cdn.worldnomads.net                     |
|     1 | cdn.adonads.com                         |
|     1 | freevideodownloader.co                  |
|     1 | boredomfiles.com                        |
|     1 | cdn.ave.zone                            |
|     1 | s.ave.zone                              |
|     1 | s.zkcdn.net                             |
|     1 | user.fasttechcdn.com                    |
|     1 | www.raptu.com                           |
|     1 | files.mpic.ws                           |
|     1 | images.mpic.ws                          |
|     1 | cdn.leagueoflegends.com                 |
|     1 | www.clicken.us                          |
|     1 | cofffre.com                             |
|     1 | marriott.btttag.com                     |
|     1 | cs.luckyorange.net                      |
|     1 | ping.luckyorange.net                    |
|     1 | www.upcload.com                         |
|     1 | ptr.wowdb.com                           |
|     1 | motorcycle.com.vsassets.com             |
|     1 | genius.com                              |
|     1 | assets.genius.com                       |
|     1 | console.brax-cdn.com                    |
|     1 | cdntrf.com                              |
|     1 | d.facdn.net                             |
|     1 | betseyjohnson.btttag.com                |
|     1 | cdnnetwok.xyz                           |
|     1 | api.adquality.ch                        |
|     1 | cfa.2mdnsys.com                         |
|     1 | static.2mdnsys.com                      |
|     1 | stats.2mdnsys.com                       |
|     1 | www.ti553.com                           |
|     1 | www.rapidvideo.com                      |
|     1 | lbstatic.nu                             |
|     1 | biddr.brealtime.com                     |
|     1 | ilosm.cdnize.com                        |
|     1 | img-fanatik.mncdn.com                   |
|     1 | cdn.ay.gy                               |
|     1 | www.liveadexchanger.com                 |
|     1 | cdn.navdmp.com                          |
|     1 | pixel.yabidos.com                       |
|     1 | option.boldapps.net                     |
|     1 | rm.boldapps.net                         |
|     1 | assets.backinstock.org                  |
|     1 | loader.wisepops.com                     |
|     1 | videowing.me                            |
|     1 | videozoo.me                             |
|     1 | www.cyberghostvpn.com                   |
|     1 | s.yumm.ly                               |
|     1 | media-dominaria.cursecdn.com            |
|     1 | static-dominaria.cursecdn.com           |
|     1 | media-hearth.cursecdn.com               |
|     1 | static-waterdeep.cursecdn.com           |
|     1 | media-waterdeep.cursecdn.com            |
|     1 | cdn.embedly.com                         |
|     1 | cdnsure.com                             |
|     1 | www.dumpert.nl                          |
|     1 | icontent.us                             |
|     1 | cdn.mathjax.org                         |
|     1 | cdn.doublepimpssl.com                   |
|     1 | events.jotform.com                      |
|     1 | 8video.tv                               |
|     1 | uploads.tapatalk-cdn.com                |
|     1 | static.betazeta.com                     |
|     1 | img2.thejournal.ie                      |
|     1 | f3.thejournal.ie                        |
|     1 | app.viralsweep.com                      |
|     1 | static.kameleoon.com                    |
|     1 | api.woosmap.com                         |
|     1 | media.nextopia.net                      |
|     1 | cdn.embed.ly                            |
|     1 | o1.t26.net                              |
|     1 | kiwiirc.com                             |
|     1 | ssl.luckyorange.com                     |
|     1 | static-strawpoll.cursecdn.com           |
|     1 | rdata.reembed.com                       |
|     1 | cdn.reembed.com                         |
|     1 | api.flattr.com                          |
|     1 | www.saznajnovo.com                      |
|     1 | display.mybroadband.co.za               |
|     1 | poll.cityzenapp.us                      |
|     1 | in2.zog.link                            |
|     1 | www.txxx.com                            |
|     1 | cdn.thegrid.io                          |
|     1 | hurriyetemlak.api.sociaplus.com         |
|     1 | static-3.nexusmods.com                  |
|     1 | static-1.nexusmods.com                  |
|     1 | static-4.nexusmods.com                  |
|     1 | staticdelivery.nexusmods.com            |
|     1 | js.gdicdn.com                           |
|     1 | css.gdicdn.com                          |
|     1 | images.gdicdn.com                       |
|     1 | i-6uf0utvje8gy-cdn.plushcontent.com     |
|     1 | static.publicocdn.com                   |
|     1 | comunique.publicocdn.com                |
|     1 | imagens.publicocdn.com                  |
|     1 | pool.distilled.ie                       |
|     1 | f1.thejournal.ie                        |
|     1 | s.tvp.pl                                |
|     1 | cdn.pushcrew.com                        |
|     1 | partner-infallible.men                  |
|     1 | pstatic.bestpriceninja.com              |
|     1 | cdnvalid.xyz                            |
|     1 | takethatad.com                          |
|     1 | us-tracker.inside-graph.com             |
|     1 | static.lystit.com                       |
|     1 | static.adf.ly                           |
|     1 | adf.ly                                  |
|     1 | zen.herolens.com                        |
|     1 | uploads.ungrounded.net                  |
|     1 | integrations.fitanalytics.com           |
|     1 | images.vinted.net                       |
|     1 | s22-de-babies.vinted.net                |
|     1 | s21-de-babies.vinted.net                |
|     1 | s20-de-babies.vinted.net                |
|     1 | tracking.g2crowd.com                    |
|     1 | www.refersion.com                       |
|     1 | m.agar.io                               |
|     1 | tag.imonomy.com                         |
|     1 | imgr.rgcdn.nl                           |
|     1 | b.bench-live.men                        |
|     1 | p.bench-live.men                        |
|     1 | www.trustl.life                         |
|     1 | wall.adgaterewards.com                  |
|     1 | www.ekranet.com                         |
|     1 | cdn.quantummetric.com                   |
|     1 | c3.nrostatic.com                        |
|     1 | c6.nrostatic.com                        |
|     1 | c1.nrostatic.com                        |
|     1 | c7.nrostatic.com                        |
|     1 | c10.nrostatic.com                       |
|     1 | c9.nrostatic.com                        |
|     1 | c8.nrostatic.com                        |
|     1 | c0.nrostatic.com                        |
|     1 | c4.nrostatic.com                        |
|     1 | c5.nrostatic.com                        |
|     1 | c2.nrostatic.com                        |
|     1 | optassets.ontraport.com                 |
|     1 | liveadexchanger.com                     |
|     1 | diariodenoticias.os.tc                  |
|     1 | vkpass.com                              |
|     1 | cdn30.vkpass.com                        |
|     1 | static.shareasale.com                   |
|     1 | w.likebtn.com                           |
|     1 | thevid.net                              |
|     1 | www.jvzoo.com                           |
|     1 | static1.sw-cdn.net                      |
|     1 | jobstreetasia.api.sociaplus.com         |
|     1 | staticssl.ibsrv.net                     |
|     1 | cdn.ibautomotive.com                    |
|     1 | shortem.com                             |
|     1 | epiclaunch.clickfunnels.com             |
|     1 | secure.optibuymac.com                   |
|     1 | www.patreon.com                         |
|     1 | netcheckcdn.xyz                         |
|     1 | mycloud.to                              |
|     1 | t.facdn.net                             |
|     1 | a.facdn.net                             |
|     1 | cdn.feathr.co                           |
|     1 | cdn.dep-x.com                           |
|     1 | bonanzascore.men                        |
|     1 | z2jakqoshx.com                          |
|     1 | cdn.vor.us                              |
|     1 | squadt.men                              |
|     1 | static.x-plarium.com                    |
|     1 | ads.summitnewsstand.com.ph              |
|     1 | stat-s9.ms-online.pl                    |
|     1 | stat-s2.ms-online.pl                    |
|     1 | stat-s8.ms-online.pl                    |
|     1 | stat-s6.ms-online.pl                    |
|     1 | stat-s1.ms-online.pl                    |
|     1 | stat-s7.ms-online.pl                    |
|     1 | stat-s4.ms-online.pl                    |
|     1 | stat-s5.ms-online.pl                    |
|     1 | stat-s3.ms-online.pl                    |
|     1 | stat-s0.ms-online.pl                    |
|     1 | stat-m2.ms-online.pl                    |
|     1 | checkout-sandbox.getbread.com           |
|     1 | checkout.getbread.com                   |
|     1 | samsungus.btttag.com                    |
|     1 | tsyndicate.com                          |
|     1 | b.top-drawer-c.life                     |
|     1 | vidup.me                                |
|     1 | tvad.me                                 |
|     1 | datawrapper.dwcdn.net                   |
|     1 | vidnode.net                             |
|     1 | coincap.io                              |
|     1 | game.gamemonkey.org                     |
|     1 | top-drawer-x.life                       |
|     1 | static.hitta.se                         |
|     1 | api.recurly.com                         |
|     1 | cdn.thetrackernetwork.com               |
|     1 | notifications.thetrackernetwork.com     |
|     1 | storage-w.rgcdn.nl                      |
|     1 | www.share-online.biz                    |
|     1 | www.serverwolves.com                    |
|     1 | 39.img.avito.st                         |
|     1 | 29.img.avito.st                         |
|     1 | 03.img.avito.st                         |
|     1 | clickcdn.shareaholic.com                |
|     1 | orcid.uservoice.com                     |
|     1 | sp.tbxnet.com                           |
|     1 | code.tinypass.com                       |
|     1 | cdn2.leisure-nb.net                     |
|     1 | weshare.me                              |
|     1 | k-vid.net                               |
|     1 | preview.ibb.co                          |
|     1 | academysports.btttag.com                |
|     1 | chatidcdn.com                           |
|     1 | www.ani-stream.com                      |
|     1 | promo.bitmedia.io                       |
|     1 | livenewschat.os.tc                      |
+-------+-----------------------------------------+
```
Attached list without the counts:
[cloudflare.txt](https://github.com/EFForg/privacybadger/files/1325521/cloudflare.txt)

**Comment by ghostwords** [comment]
The same list but grouped by version:
```
+-------+-------------+
| count | version     |
+-------+-------------+
|    43 | 2017.9.12.1 |
|    85 | 2017.9.12   |
|   626 | 2017.7.24   |
|   671 | 2017.6.13.1 |
|     8 | 2017.6.13   |
|  1096 | 2017.5.9    |
+-------+-------------+
```
Just the Medium domains grouped by version:
```
+-------+-------------+
| count | version     |
+-------+-------------+
|     5 | 2017.9.12   |
|    80 | 2017.7.24   |
|   155 | 2017.6.13.1 |
|     2 | 2017.6.13   |
|   381 | 2017.5.9    |
+-------+-------------+
```

**Comment by ghostwords** [comment]
Should next see which (of the top reported ones) are only here because of `__cfduid` cookies (and are causing site breakages).

**Comment by ghostwords** [comment]
`edge.alluremedia.com.au` is one. New Badger don't seem to learn to block it, but old Badgers still run into issues; error report counts by month of reports where this domain is blocked:
```
+---------+----------+
| ym      | count(*) |
+---------+----------+
| 2017-10 |        1 |
| 2017-09 |        9 |
| 2017-08 |       14 |
| 2017-07 |       19 |
| 2017-06 |       21 |
| 2017-05 |       25 |
| 2017-04 |       39 |
| 2017-03 |       16 |
| 2017-02 |       22 |
| 2017-01 |       15 |
| 2016-12 |       11 |
| 2016-11 |       13 |
| 2016-10 |       15 |
| 2016-09 |        9 |
| 2016-08 |        9 |
| 2016-07 |        6 |
| 2016-06 |        4 |
| 2016-05 |        3 |
| 2016-04 |       10 |
| 2016-03 |       17 |
| 2016-02 |       16 |
| 2016-01 |       12 |
| 2015-12 |       14 |
| 2015-11 |       15 |
| 2015-10 |       13 |
| 2015-09 |       11 |
| 2015-08 |       10 |
| 2015-07 |        1 |
+---------+----------+
```
We released #1361 with Privacy Badger version 2017.5.9. I guess affected users either whitelisted the affected site, unblocked the relevant domain, or uninstalled Privacy Badger.
